live-examples/wat-examples/statements/select.wat

```diff
@@ -1,15 +1,18 @@
 (module
-  (import "console" "log" (func $log (param i32)))
-  (func $main
+  (func (export "select_10_or_20") (result i32)
```

Comment:
Maybe rename the function something like `select_simple` and `select_externref`?